WeightProducer/python/MCSampleValues.py

```diff
@@ -251,744 +251,497 @@ class MCSampleValuesHelper():
         },
         "TTTo2L2Nu" : {
             "CrossSection" : XSValues(
-                XS_13TeV=88.29, XSSource_13TeV="XSDB (NNLO)",
+                XS_13TeV=87.34, XSSource_13TeV="PDG XS - https://pdg.lbl.gov/2021/reviews/rpp2020-rev-top-quark.pdf",
             ),
         },
-        "TTToHadronic" : {
-            "CrossSection" : XSValues(
-                XS_13TeV=377.96, XSSource_13TeV="XSDB (NNLO)",
+        "TTTo2L2Nu_mtop166p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=811.4, XSSource_13TeV="XSDB (NLO)",
```

Comment:
How is this cross section, `811.4`, so many times higher than the `TTTo2L2Nu` cross section of `87.34`?

Reply:
Made the top mass variations behave like the nominal samples---by using branching fraction. Also implemented a k factor. 33f4d81